prime-dotnet-webapi/Models/EnrolleeAbsence.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Newtonsoft.Json;
+
+namespace Prime.Models
+{
+  [Table("EnrolleeAbsence")]
+  public class EnrolleeAbsence : BaseAuditable, IEnrolleeNavigationProperty
+  {
+    [Key]
+    public int Id { get; set; }
+
+    [JsonIgnore]
+    public int EnrolleeId { get; set; }
+
+    [JsonIgnore]
```

Comment:
Since this is a new, rather isolated model that uses View Models correctly; I feel its worth starting to avoid using `[JsonIgnore]` in our Models. Its a hold-over from the before times when we were exposing our domain models directly. The only caveat: if this is supposed to be part of the enrolleee snapshot we might have to use these headers anyway :(